api/UnitTest/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+using Xunit;
```

Comment:
Could we rename this file to something more descriptive. Like ExampleUnitText.cs?